src/command.coffee

```diff
@@ -48,7 +48,8 @@ SWITCHES = [
   ['-r', '--require [MODULE*]', 'require the given module before eval or REPL']
   ['-s', '--stdio',             'listen for and compile scripts over stdio']
   ['-l', '--literate',          'treat stdio as literate style coffeescript']
-  ['-t', '--tokens',            'print out the tokens that the lexer/rewriter produce']
+  ['-t', '--transpile',         'pipe generated JavaScript through Babel']
```

Comment:
We need to add this to the docs as well, right? Or is docs stuff done some other time?

Do we need to document the "breaking change" that `-t` now means `--transpile` instead of `--tokens`?

Reply:
I’m working on the docs, I think I’ll add the docs changes to this PR once I finish getting your feedback (like if you thought I should use `--babel` or something).

I don’t know, do you think anyone cares that I’m reassigning `-t`? Is that really a breaking change? Perhaps it’s small enough that I mention it only in the changelog?

Reply:
Let’s just mention it in the changelog.